framework/src/node/consensus/consensus.ts

```diff
@@ -203,7 +193,7 @@ export class Consensus {
 	}
 
 	public finalizedHeight(): number {
-		return this._bft.finalizedHeight;
+		return 0;
```

Comment:
why 0 🤔 ?

Reply:
To implement this we need bft module, but ill add use the empty interface here to mimic what we have